src/common/hooks/transaction/use-edit-postconditions.ts

```diff
@@ -1,13 +1,14 @@
+// @ts-nocheck
```

Comment:
🤔

Reply:
This file is not currently used due to it previously being broken. My next PR will re-implement all the edit postcondition stuff